README.md

```diff
@@ -26,8 +26,8 @@ You can experience the capabilities of Asgardio .NET OIDC SDK by following this
 
 ### Prerequisites
 1. Microsoft Windows 8 (Or server equivalent) or greater.
-2. [.NET Framework Standard 4.6.1](https://dotnet.microsoft.com/download/dotnet-framework/net461) or greater.
-3. WSO2 Identity Server
+2.[.NET Framework Standard 4.6.1](https://dotnet.microsoft.com/download/dotnet-framework/net461) or greater.
+3.[ WSO2 Identity Server](https://wso2.com/identity-and-access-management/)
```

Comment:
Keep a space before the beginning of the square bracket and remove the space after the start of the square bracket